great_expectations/expectations/metrics/map_metric_provider/column_condition_partial.py

```diff
@@ -189,6 +189,7 @@ def inner_func(
                     **metric_value_kwargs,
                     _dialect=dialect,
                     _table=selectable,
+                    _execution_engine=execution_engine,
```

Comment:
This pattern can be extended to other condition partials (passing the execution engine object).